src/components/Scanner.vue

```diff
@@ -1,44 +1,86 @@
 <template>
   <ion-toolbar>
-    <ion-buttons slot="end" @click="closeScanner()" >
-      <ion-button >
+    <ion-buttons slot="end" @click="closeScanner()">
+      <ion-button>
         <ion-icon :icon="closeOutline" />
       </ion-button>
-    </ion-buttons>  
-  </ion-toolbar>   
+    </ion-buttons>
+  </ion-toolbar>
+
   <div class="scanner">
+    <!-- Conditionally render the "Start Scanning" button based on camera access -->
+    <ion-button v-if="!hasCameraAccess" @click="openScanner">Start Scanning</ion-button>
```

Comment:
We do not need to make any change in this component, because once we have added the check of camera access in the parent component then this component will not open and thus there is no use of having any checks in this component.